packages/office-addin-dev-settings/src/commands.ts

```diff
@@ -179,6 +179,21 @@ export async function getSourceBundleUrl(manifestPath: string) {
   }
 }
 
+async function getWebViewType(manifestPath: string) {
+  try {
+    const manifest = await readManifestFile(manifestPath);
+
+    validateManifestId(manifest);
+
+    const currentWebViewString = await devSettings.getWebView(manifest.id!);
+    currentWebViewString ? console.log("The webViewType is set to " + currentWebViewString + ".") : 
```

Comment:
You could simplify the name to just `webViewString`. 

Why not have return the WebViewType enum. You could write a function in devSettings toWebViewTypeString(webViewType: WebViewType): string   which uses the switch to return the string constant for the enum value. For Default, you can return an empty string.

Then here in the command you could use the function to convert to the display text and show it in the message.

Use `${variable} in a string` form rather than using + to concatenate the string.

Make the string contants mixed case, e.g. "Edge" so they display nicer. The case shouldn't matter for the command input.

Reply:
When running getWebView(), we read what's in the registry WebView selection for the addin and return that string back. There isn't an enum that gets passed back in the first place, so I don't see the benefit of adding a string-to-enum conversion when we first get this result, only to later have an enum-to-string conversion in the command. 

That being said, the `${variable}` is a good point, and making the strings mixed case should be done. I've modified the constants we're writing to the registry to be mixed case, so it gets returned as mixed case to the user.

Reply:
The point of converting the string to the enum value is so that the function works on enum values. The conversion from the enum value to a string is so the command can show the string. 

Having the exported function return the enum makes for a better "api" for using the package from code.

Reply:
Would the recent way I've modified getWebViewType() be an appropriate way to make sure an enum is returned with the exported webViewOverride()?